### DISCUSSION TITLE
Using sa_column with onupdate in parent model leads to sqlalchemy.exc.ArgumentError:

### DISCUSSION THREAD

**Discussion by Igralino** [fastapi/sqlmodel#754] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from datetime import datetime
from typing import Optional

from sqlalchemy import Column, DateTime, func
from sqlalchemy.orm import declared_attr
from sqlmodel import SQLModel, Field


class BaseModel(SQLModel):
    @declared_attr
    def __tablename__(cls) -> str:
        return cls.__name__.replace("Model", "").lower() + "s"

    created_at: datetime = Field(default_factory=datetime.now, nullable=False)
    updated_at: Optional[datetime] = Field(sa_column=Column(DateTime(), onupdate=func.now()))

class UserModel(BaseModel, table=True):
    id: str = Field(primary_key=True, min_length=1, max_length=200)
    name: str = Field(max_length=200, min_length=1)
```

### Description

- I've created a BaseModel with `updated_at: Optional[datetime] = Field(sa_column=Column(DateTime(), onupdate=func.now()))` field (took it from https://github.com/tiangolo/sqlmodel/issues/594)
- Then I made an ancestor of the BaseModel - the UserModel
- Then I run `alembic revision --autogenerate` and it falls with error
`sqlalchemy.exc.ArgumentError: Column object 'updated_at' already assigned to Table 'users'`

I saw https://github.com/tiangolo/sqlmodel/discussions/743, but it is not my case, because I want to use `onupdate` feature

### Operating System

macOS

### SQLModel Version

0.0.14

### Python Version

3.12.1

### Additional Context

Alembic version: 1.13.0

**Comment by jp-u** [comment]
Ran into the same issue, however, it has only arisen once I've attempted to inherit an additional table model from the `BaseModel` (ie `CarModel(BaseModel, table=True)`

**Comment by snehith-kintsugi** [comment]
Did you figure out a solution for this?

**Reply by Igralino** [reply]
Nope 😢

**Comment by joachimhuet** [comment]
Hey  there,

Head over there: https://github.com/tiangolo/sqlmodel/discussions/876#discussioncomment-9240716 to get an answer to your question.

The problem is in using sqlalchemy `Column`, try using `sa_column_args` and `sa_column_kwargs` instead 😄
